@AH-64D Apache Longbow/Addons/fza_ah64_audioSystem/functions/fn_addCaution.sqf

```diff
@@ -0,0 +1,25 @@
+/* ----------------------------------------------------------------------------
+Function: fza_audioSystem_fnc_addCaution
+
+Description:
+
+Parameters:
+    _heli: object - the apache the player is in
+
+Returns:
+    No returns
+
+Examples:
+    [_heli] call fza_audioSystem_fnc_addCaution;
+Author:
+    Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params["_heli"];
+
+_heli setvariable ["fza_audio_caution", true];
+
+private _FuncHook = _heli getVariable ["fza_audio_FuncHook", scriptNull];
+if (scriptDone _FuncHook) then {
+    _FuncHook = [_heli] spawn fza_audioSystem_fnc_audioSystemHandler;
+    _heli setVariable ["fza_audio_FuncHook", _FuncHook];
```

Comment:
Please change _FunfHook to _funcHook for both the function name and variable name in lines 23 and 24.